core/src/main/kotlin/ch/ergon/dope/resolvable/clause/model/joinHint/KeysOrIndexHint.kt

```diff
@@ -15,19 +15,19 @@ import ch.ergon.dope.validtype.ArrayType
 import ch.ergon.dope.validtype.StringType
 import ch.ergon.dope.validtype.ValidType
 
-interface UseKeysOrIndexHint : Resolvable
+interface KeysOrIndexHint : Resolvable
 
-class UseKeysHintClass private constructor(
+class KeysHintClass private constructor(
     private val useKeys: TypeExpression<out ValidType>,
-) : UseKeysOrIndexHint {
+) : KeysOrIndexHint {
     companion object {
         @JvmName("singleUseKeysHintConstructor")
```

Comment:
Please remove the Use from the JvmNames as well (same for the constructor below)

Reply:
done